src/sentry/processing/realtime_metrics/redis.py

```diff
@@ -80,3 +84,59 @@ def increment_project_duration_counter(
             pipeline.hincrby(key, duration, 1)
             pipeline.pexpire(key, self._histogram_ttl)
             pipeline.execute()
+
+    def get_lpq_projects(self) -> Set[int]:
+        """
+        Fetches the list of projects that are currently using the low priority queue.
+
+        Returns a list of project IDs.
+        """
+        return set(
+            filter(
+                None,
+                {
+                    _to_int(project_id_raw)
+                    for project_id_raw in self.cluster.smembers(LPQ_MEMBERS_KEY)
+                },
+            )
+        )
+
+    def add_project_to_lpq(self, project_id: int) -> None:
+        """
+        Moves a project to the low priority queue.
+
+        This forces all symbolication events triggered by the specified project to be redirected to
+        the low priority queue, unless the project is manually excluded from the low priority queue
+        via the `store.symbolicate-event-lpq-never` kill switch.
```

Comment:
i've removed all of the bits that talk about what other functions are doing (e.g. `should_demote_..` in preprocessing), and made the language less assertive about what it does, since the presence of a project in this list does not imply that it's guaranteed to be in the LPQ.